src/__tests__/deleteMessage.test.ts

```diff
@@ -0,0 +1,22 @@
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import * as apiClient from '../';
+import { deleteMessage, invalidDeleteMessage } from './fixtures/messages';
+
+const mock = new MockAdapter(axios);
+
+afterEach(() => {
+	mock.reset();
+});
+
+const apiClientInit = new apiClient.APIClient({ apiBase: 'unics-social' });
+
+test('deleteMessage(): deletes message correctly', async () => {
+	mock.onDelete(`unics-social/channels/${deleteMessage.channelID}/messages/${deleteMessage.messageID}`).reply(200);
+	await apiClientInit.deleteMessage(deleteMessage);
```

Comment:
This test could use an assertion - maybe you could check that it returns undefined?